app/models/forms/export.rb

```diff
@@ -8,6 +8,45 @@ class Export
       DisplayLogic DisplayConditions Default Hidden
     ].freeze
 
+    OPERATIONS = {
+      "eq" => "=",
+      "neq" => "!=",
+      "lt" => "<",
+      "leq" => "<=",
+      "gt" => ">",
+      "geq" => ">="
+    }.freeze
```

Comment:
it looks like there's an existing `ODK::ConditionDecorator.OP_XPATH` we can use here to DRY up the code a bit. it also happens to include `ninc` in case that's another option we need to handle.